src/main/java/goodgenerator/blocks/tileEntity/LargeEssentiaGenerator.java

```diff
@@ -84,19 +83,9 @@ public boolean checkMachine_EM(IGregTechTileEntity aBaseMetaTileEntity, ItemStac
         mEssentiaHatch.clear();
         return structureCheck_EM(mName, 4, 0, 4) && (mDynamoHatches.size() + eDynamoMulti.size()) == 1
                 && checkHatchTier()
-                && checkNoLaser()
```

Comment:
Like chochom said, Lasers are a no go, no exceptions, indefinitely, the higher ups have already made this clear

Reply:
Now that the cost of LEG has been greatly increased, please, can we forget about the crumbling balance of a few months ago to think about the viability of this new balance? Now with lasers, there's no problem, LEGs can't outperform any mainstream generator even with lasers, after this new series of balance adjustments (basically all nerfs)